src/components/Chat/index.css

```diff
@@ -0,0 +1,97 @@
+main {
```

Comment:
Can you give classNames to this css file, this way it’ll affect all the elements on the website

Reply:
PTAL